### ISSUE TITLE
On first load I see a list of old tasks

### ISSUE THREAD

**Issue by sglangevin** [medic/medic-webapp#1995]
First load of a restricted user on alpha.dev, some tasks load but they are old tasks, not current, and there are no icons. Icons and my actual list of tasks appears only after clicking Update.

**Comment by garethbowen** [comment]
@sglangevin Do you mean "Update" in the modal? It could be that that update has pulled down new task configuration or icons in which case this is exactly what I'd expect to happen. Can you try a clean browser?

**Comment by sglangevin** [comment]
This was on a clean browser, first load. I meant Update on the About page.

**Comment by garethbowen** [comment]
It's so confusing that the link on the about page is labelled as "Update" for LG. It should be "Refresh" or "Reload" as it doesn't actually _do_ anything.

**Comment by garethbowen** [comment]
@sglangevin What server/app version was this on? #1969 looks related and was fixed very recently.

**Comment by sglangevin** [comment]
alpha.dev version 2866

**Comment by garethbowen** [comment]
That should have contained the fix for #1969 so this definitely needs some attention.

**Comment by garethbowen** [comment]
I think the icons and old tasks are separate issues. The icons is probably due to the resources doc where the icons are stored not existing initially, and nothing is watching for when it gets updated. This will also mean any changes to icons won't show up on devices until the user reloads.

The tasks changing is different. It may be a configuration issue. I'll keep looking but I think it's different to the icon issue.

**Comment by garethbowen** [comment]
I've split the icon bit off into #1997 leaving this one to work out why the wrong tasks were rendering.

**Comment by garethbowen** [comment]
My initial thought is that some tasks on alpha.dev might not be being emitted with `resolved:false` so there's no event to update the UI.

Either that or replication isn't working as well as it should be...

**Comment by garethbowen** [comment]
I suspect this is related to #1996 - replication is either failing or too slow.

**Comment by garethbowen** [comment]
Closing for Acceptance Testing - this should be fixed by the configuration changes.
